app/src/main/res/layout/add_profile_activity.xml

```diff
@@ -64,16 +64,33 @@
             app:layout_constraintEnd_toEndOf="@+id/add_profile_activity_user_image_view"
             app:layout_constraintRight_toRightOf="@+id/add_profile_activity_user_image_view" />
 
-          <TextView
-            android:id="@+id/add_profile_activity_required_heading_text_view"
-            android:layout_width="match_parent"
+          <LinearLayout
+            android:id="@+id/add_profile_activity_required_relative_layout"
```

Comment:
Rename to `add_profile_activity_required_text_container`

Reply:
Now we only have a text_view its not required